assets/chezmoi.io/docs/reference/templates/functions/isExecutable.md

```diff
@@ -0,0 +1,11 @@
+# `isExecutable` *file*
+
+`isExecutable` returns true if a file is executable.
+
+!!! example
+
+    ```
+        {{ if isExecutable "/bin/echo" }}
+            `sh` is executable
```

Comment:
This should be `echo is executable`. You should also remove the backticks around sh as this is text or shell, not Markdown.

Reply:
Yes you're correct on both accounts. I would fix it but I think I missed the boat on that.